requirements.txt

```diff
@@ -1,2 +1,3 @@
 slackclient>=2.0.0
 requests
+prometheus_client
```

Comment:
nit: alphabetical ordering?  Probably won't grow that long, but it makes it easier to find entries if this does become a long list.

Reply:
updated!